step-capstone/src/components/Flight.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Box, Grid, Typography } from '@material-ui/core'
+
+/*
+Takes data in this form:
```

Comment:
Where does this json come from? If it's a source where there's possible errors, consider adding a JSON schema (https://www.npmjs.com/package/jsonschema) so that you can validate the input. The schema then serves the same documentation purpose as your comment does now.

Reply:
That makes sense! The data should be passed down directly from the database in a higher level component.